django/urls/converters.py

```diff
@@ -0,0 +1,48 @@
+from django.conf import settings
+from django.utils import lru_cache
+
+
+class BaseConverter(object):
+    pass
+
+
+class IntConverter(BaseConverter):
+    regex = '[0-9]+'
```

Comment:
Most URLs with numbers are for identifiers (primary keys), and as such not allowing negative numbers makes sense. On the other hand, I do agree with the some of the remarks in the related Flask thread, especially

"Regarding negative numbers, the most popular usecase is to allow only positive numbers. Calling those converters int might have been a mistake though".

The question is: do we think it was a mistake to not allow negative numbers for the `int` converter?